backend/main.py

```
@@ -2041,16 +2038,23 @@ def get_validation_stats():
 
 # ==================== LEGACY PERSONA ENDPOINTS ====================
 
-@app.route('/api/ai-firm/personas/warren', methods=['POST'])
+@app.route('/api/ai-firm/personas/warren', methods=['GET', 'POST'])
 def warren_analysis():
     """Warren Persona Analysis Endpoint"""
     if not AI_FIRM_READY: return jsonify({'error': 'offline'}), 500
```

Flake8 (7.3.0)

[error] multiple statements on one line (colon)

(E701)